File: project2/app/src/main/java/com/minkiapps/project2/MainActivity.java
package com.minkiapps.project2;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

import com.minkiapps.project2.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);

        binding.tvActMain.setText("HELLO THERE!");
    }
}

File: project1/app/src/main/java/com/minkiapps/project1/MainActivity.kt
package com.minkiapps.project1

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.minkiapps.projectlibrary.PiSolver

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        PiSolver().solvePi()
    }
}